PrestaShop/PrestaShop
# [-] BO : introduced a reusable way to display a right sidebar

## Description

We want to be able to act on a right sidebar to display related informations.
For instance, on product catalog we can quickly move to another product throught the product list sidebar.

![quick_nav](https://cloud.githubusercontent.com/assets/1247388/13878426/9b378652-ed11-11e5-8bdf-b9ec390342ef.png)

We want to use this system to make the docs always available anywhere, and related to the page the e-merchandizer actualy visits. (like in 1.6)

The new migrated pages have an inconsistent behavior on this point: this contribute is about put the docs the right way, using a reusable sidebar system.
## Steps to Test this Fix
1. Step 1.
   Visit all Product/Module pages
2. [x] _Product_ catalog
3. [x] _Product_ form
4. [x] _Modules_ catalog
5. [x] _Modules_ installed modules
6. [x] _Modules_ notifications
7. Step 2.
   Click on the help button in the top bar
8. Step 3.
   You should see a sidebar displayed

![products_help](https://cloud.githubusercontent.com/assets/1247388/13878512/39eeada2-ed12-11e5-87ec-401cec9dde2b.png)
## Forge Ticket (optional)

This improvement will fix http://forge.prestashop.com/browse/BOOM-238 and make the Back Office easier to maintain.
